src/com/company/Veterinary.java

```diff
@@ -28,14 +28,15 @@ public void showAll (){
             }
         }
     }
-    public void remove(int id){
+    public boolean remove(int id){
         for(int i =0 ; i< pets.size(); i++){
             if(pets.get(i).getId() == id )
             {
```

Comment:
@Vanessa-Sanjinez please take care about curly braces be sure that you are keep the same syntax